### DISCUSSION TITLE
QTree selects children items from different nodes if their label is the same

### DISCUSSION THREAD

**Discussion by Nikjee** [quasarframework/quasar#15496] [General - Components / Directives / etc] [answered]
The problem is if multiple nodes have a children with a same label and you click on one of them then all the same labels getting ticked. 

In picture below i clicked on first "Good recipe" and second one got ticked aswell.

I don't know if its a usual behaviour for leaf strategy but i would like to know if there a fix for this?

<img width="344" alt="image" src="https://user-images.githubusercontent.com/35499368/221433379-339f4d64-7450-4ab3-8945-f33ea5e92744.png">

**Comment by metalsadman** [comment, marked as answer]
Because you are using label as your node-key, provide a unique one to avoid such behavior.

**Reply by Nikjee** [reply]
Thanks that worked!
